blamo/lib/Unit/unit.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:blamo/main.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+import 'package:intl/intl.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class UnitPage extends StatefulWidget {
+  StateData pass; //Passes the StateData object to the stateful constructor
+
+  UnitPage(this.pass);
+
+  @override
+  _UnitPageState createState() => new _UnitPageState(pass);
+}
+
+class _UnitPageState extends State<UnitPage> {
+  final routeName = '/TestPage';
+  StateData currentState;
+  _UnitPageState(this.currentState);
+
+@override
+  final GlobalKey<FormBuilderState> _fbKey = GlobalKey<FormBuilderState>();
+  Widget build(BuildContext context) {
+    return new Scaffold(
+        backgroundColor: Colors.white,
+        drawer: new Drawer(
+        child: SideMenu(currentState)
+      ),
+        appBar: new AppBar(
+            title: new Text("Log Info Page"),
```

Comment:
This text should be changed to test page for the top bar.

Reply:
updated